common/src/WooCommerce.php

```diff
@@ -14,6 +15,8 @@ public function __construct(BasePlugin $plugin) {
         $this->plugin = $plugin;
         add_action('woocommerce_order_status_completed', [$this, 'order_completed'], 10, 1);
         add_action('woocommerce_checkout_update_order_meta', [$this, 'set_order_language']);
+	    add_action('woocommerce_product_options_sku', [$this, 'gtin_wwk_option']);
+	    add_action('woocommerce_admin_process_product_object', [$this, 'save_wwk_product_option']);
```

Comment:
Tabs to spaces pls.

Reply:
And if you add the editorconfig file that I mentioned above, your editor wouldn't fuck this up.

Reply:
ah sorry, I edited `editorconfig` but then I added `wordpress` to the framework library in PhpStorm, and this change it back again. 

I will make sure to it does not happen again.